open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/KafkaVirtualiser.java

```diff
@@ -2,13 +2,21 @@
 /* Copyright Contributors to the ODPi Egeria project. */
```

Comment:
Should not have any Kafka specific code in the virtualizer - should all be handled by the connector